Dockerfile

```diff
@@ -17,6 +17,16 @@ RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -tags strictfipsruntime -a -o
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.8
 WORKDIR /
+
+RUN microdnf update -y && \
+    microdnf install tar -y \
+    && microdnf clean all && \
+    rm -rf /var/cache/yum
+ADD https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz /tmp/openshift-client-linux.tar.gz
```

Comment:
use this instead please https://gitlab.cee.redhat.com/data-hub/rhods-cpaas-midstream/-/blob/rhods-1.34-rhel-8/distgit/containers/odh-deployer/Dockerfile.in?ref_type=heads

Reply:
I'm trying to pull `registry.redhat.io/openshift4/ose-cli:v4.13` but seem to get a permission error from registry.redhat.io. Looking into it.